Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Use the official Ruby 3.2 image as a base image
+ARG RUBY_VERSION=3.2
```

Comment:
I figure if someone is using a particular version of Ruby they can change change it here.